client/blocks/eligibility-warnings/hold-list.jsx

```diff
@@ -24,63 +24,64 @@ function getHoldMessages( translate ) {
 		},
 		TRANSFER_ALREADY_EXISTS: {
 			title: translate( 'Installation in progress' ),
-			description: translate( 'Another installation is already in progress.' ),
-			supportUrl: 'https://wordpress.com/help'
+			description: translate( 'Please wait for the other installation to complete, then try again.' ),
+			supportUrl: 'https://wordpress.com/help',
 		},
+		// Handled as a banner
 		NO_BUSINESS_PLAN: {
 			title: translate( 'Business plan required' ),
 			description: translate( 'This feature is only allowed on sites with a business plan.' ),
-			supportUrl: 'https://support.wordpress.com/'
+			supportUrl: 'https://support.wordpress.com/',
 		},
 		NO_JETPACK_SITES: {
-			title: translate( 'Jetpack site' ),
-			description: translate( 'This feature is not supported on Jetpack sites.' ),
-			supportUrl: 'https://wordpress.com/help'
+			title: translate( 'Jetpack not supported' ),
+			description: translate( 'Try using a different site.' ),
 		},
 		NO_VIP_SITES: {
-			title: translate( 'VIP site' ),
-			description: translate( 'This feature is not supported on VIP sites.' ),
-			supportUrl: 'https://wordpress.com/help'
+			title: translate( 'VIP site not supported' ),
+			description: translate( 'Try using a different site.' ),
 		},
 		SITE_PRIVATE: {
-			title: translate( 'Private site' ),
-			description: translate( 'This feature is not supported on private sites.' ),
-			supportUrl: 'https://support.wordpress.com/'
+			title: translate( 'Private site not supported' ),
+			description: translate( 'Make your site public to resolve.' ),
+			supportUrl: 'https://support.wordpress.com/settings/privacy-settings/',
 		},
 		SITE_GRAYLISTED: {
-			title: translate( 'Flagged site' ),
-			description: translate( 'This feature is not supported on sites that are not in good standing.' ),
-			supportUrl: 'https://wordpress.com/help'
+			title: translate( 'Flagged site not supported' ),
+			description: translate( "Contact us to review your site's standing to resolve." ),
+			supportUrl: 'https://support.wordpress.com/suspended-blogs/',
 		},
 		NON_ADMIN_USER: {
 			title: translate( 'Admin access required' ),
 			description: translate( 'Only site administrators are allowed to use this feature.' ),
-			supportUrl: 'https://support.wordpress.com/user-roles/'
+			supportUrl: 'https://support.wordpress.com/user-roles/',
 		},
+		// Handled as a banner
 		NOT_USING_CUSTOM_DOMAIN: {
 			title: translate( 'No custom domain' ),
 			description: translate( 'Your site must use a custom domain to use this feature.' ),
```

Comment:
Can we say "Your site must use a custom domain as your primary address to use this feature." The current wording is technically inaccurate and may be extra confusing for users who see this after purchasing a custom domain. I would link the user to the domains section for their site rather than the support page. So `/domains/manage/:site`